Reply/app/src/main/java/com/materialstudies/reply/ui/home/HomeFragment.kt

```diff
@@ -76,6 +79,16 @@ class HomeFragment : Fragment(), EmailAdapter.EmailAdapterListener {
         savedInstanceState: Bundle?
     ): View? {
         binding = FragmentHomeBinding.inflate(inflater, container, false)
+
+        lifecycleScope.launch {
+            AdaptiveUtils.screenSizeState.collect { size ->
+                screenSize = size
+                // When you click an email and change the Screen Size
+                if (screenSize != SMALL) {
+                    findNavController().navigateUp()
+                }
+            }
+        }
```

Comment:
Delete since we can just open and close the pane when an email is clicked and let SlidingPaneLayout do the rest when the screen size changes